source/unified-test-format/unified-test-format.rst

```diff
@@ -2649,6 +2650,14 @@ to elsewhere in the specification as the internal MongoClient.
 Determine the server version and topology type using the internal MongoClient.
 This information will be used to evaluate any future `runOnRequirement`_ checks.
 
+In addition to the cluster URI, the test runner MUST also be configurable
+with two other connection strings (or equivalent configuration) that point to
+TCP load balancers: one fronting multiple servers and one fronting a single
+server. These will be used to initialize client entities when executing tests
+against a load balanced cluster. If the topology type is ``LoadBalanced``,
+the test runner MUST error if either of these URIs is not provided. For all
+other topology types, these URIs MUST be ignored.
+
```

Comment:
In the paragraph under "Executing a Test" that contains "If createEntities is specified, the test runner MUST create each entity accordingly...", I think you can add a sentence reminding implementations to consult these connection strings when creating client entities for `load-balanced` topolgies.

```
If the topology type is `load-balanced`, client entities MUST be initialized with the appropriate load balancer connection string as discussed in `useMultipleMongoses <entity_client_useMultipleMongoses_>`_.
```

Reply:
Added with one small change: `LoadBalanced` rather than `load-balanced`. See [this comment thread](https://github.com/mongodb/specifications/pull/935/files/a264791eace83406fdb1057d6eb78670759b6fce..2a666f2108a5c6c028baac88c454d1ee786eff06#r608093800) for an explanation.